.travis/install-fusesoc.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -x
+
+mkdir -p $HOME/src/cores
+mkdir -p $HOME/src/tools
+mkdir -p $HOME/tools
+
+# Get our toolchain
+cd $HOME/src/tools
+curl --remote-name --location \
+  https://github.com/stffrdhrn/gcc/releases/download/or1k-9.0.0-20181113/or1k-elf-9.0.0-20181112.tar.xz
+tar xC $HOME/tools -f $HOME/src/tools/or1k-elf-9.0.0-20181112.tar.xz
+
+PATH="$HOME/tools/or1k-elf/bin:${PATH}"
+
+# Get iverilog latest source, the version in trusty is no good
+cd $HOME/src/tools
+curl --remote-name --location \
+  http://shorne.noip.me/downloads/verilog-10.2.tar.gz
+tar xf verilog-10.2.tar.gz
+cd verilog-10.2
+./configure
+make
+sudo make install
+
+export PATH
+
+cd $HOME/src/tools
+git clone https://github.com/openrisc/or1k-tests.git ;\
+cd or1k-tests/native
+make
+
+# Get the cores for our test to run (in the future these should be in fusesoc)
```

Comment:
This uses some deprecated syntax that will be remove. I think the rest of the file could be replaced with 
fusesoc init -y
fusesoc library add mor1kx-generic https://github.com/stffrdhrn/mor1kx-generic
fusesoc library add intgen https://github.com/stffrdhrn/intgen
I'm not sure why $TRAVIS_BUILD_DIR is added to fusesoc.conf though. Can you elaborate?

Reply:
This sounds good.

> I'm not sure why $TRAVIS_BUILD_DIR is added to fusesoc.conf though. Can you elaborate?

$TRAVIS_BUILD_DIR is the root of `mor1kx` git repo, its the current git branch under test (i.e. the PR code) already checked out onto the travis build host.  We need fusesoc to add it to the core list.

Reply:
FYI, after running it the config will be something like

```
[main]
cores_root = /home/travis/src/cores
  /home/travis/build/mor1kx/master
```

Can we just do `fusesoc library add $TRAVIS_BUILD_DIR`?

Reply:
thanks, Ill make those clean ups before merging, it should be easy.